packages/x-components/src/x-modules/facets-next/components/facets/__tests__/facets-provider.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { Facet, Filter } from '@empathyco/x-types-next';
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuex, { Store } from 'vuex';
+import { createSimpleFacetStub } from '../../../../../__stubs__/facets-stubs.factory';
+import { installNewXPlugin } from '../../../../../__tests__/utils';
+import {
+  getXComponentXModuleName,
+  isXComponent
+} from '../../../../../components/x-component.utils';
+import { XPlugin } from '../../../../../plugins/x-plugin';
+import { RootXStoreState } from '../../../../../store/store.types';
+import { arrayToObject } from '../../../../../utils/array';
+import { DeepPartial, Dictionary } from '../../../../../utils/types';
+import { DefaultFacetsService } from '../../../service/facets.service';
+import { facetsNextXModule as facetsXModule } from '../../../x-module';
+import { resetXFacetsStateWith } from '../../__tests__/utils';
+import FacetsProvider from '../facets-provider.vue';
+
+describe('testing Facets component', () => {
+  it('is an XComponent', () => {
+    const { wrapper } = renderFacetsProviderComponent();
+    expect(isXComponent(wrapper.vm)).toEqual(true);
+  });
+
+  it('has FacetsModule as XModule', () => {
+    const { wrapper } = renderFacetsProviderComponent();
+    expect(getXComponentXModuleName(wrapper.vm)).toEqual('facetsNext');
+  });
+
+  it('keeps the state facets if no facets are provided', () => {
+    const stateFacets = {
+      color_facet: createSimpleFacetStub('color', createSimpleFilter => [
```

Comment:
Use the `Next` version of these `createSimpleFacetStub` utility: `createNextSimpleFacetStub`